neuralmonkey/encoders/raw_rnn_encoder.py

```diff
@@ -11,9 +12,31 @@
 from neuralmonkey.nn.utils import dropout
 from neuralmonkey.dataset import Dataset
 
-# pylint: disable=invalid-name
-RNNCellTuple = Tuple[tf.contrib.rnn.RNNCell, tf.contrib.rnn.RNNCell]
-# pylint: enable=invalid-name
+
+# pylint: disable=too-few-public-methods
+class RNNSpec(object):
```

Comment:
Pokud tomu neplánuješ metody nebo properties, tak by to měla být typovaná `namedtuple`, pak je v pořádku, že nemá public methods.